### PR TITLE
adding gempy 

### PR DESCRIPTION
**Pull request by AndrewAnnex**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/gempy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  run:
+    - python
+    - pandas
+    - matplotlib
+    - numpy
+    - networkx
+    - pyvista
+    - scikit-image >=0.17
+    - seaborn >=0.9
+    - Theano >=1.0.4
```
Comment (npavlovikj) [inline recipes/gempy/meta.yaml:30]: Conda packages should be lower-case, so please replace this with `theano`.

**Comment by npavlovikj** [comment]
@Leguark, are you ok being listed as a maintainer in this recipe?

File: recipes/gempy/meta.yaml (hunk 1 above)
Comment (AndrewAnnex) [reply recipes/gempy/meta.yaml]: Done

**Comment by Leguark** [comment]
@npavlovikj  Yes, I am thanks

**Comment by npavlovikj** [comment]
Thank you for contributing @AndrewAnnex 🎉

**Pull request merged**